src/Pages/SettingsPage/SettingsPage.jsx

```diff
@@ -56,24 +84,24 @@ const SettingsPage = () => {
                     onChange={(e) => {
                       setDetails({ ...details, password: e.target.value });
                     }}
+                    value={details.password}
                   />
                 </div>
               </div>
               <div className="p-2 w-1/2">
                 <div className="relative">
                   <label
-                    htmlFor="username"
+                    htmlFor="Banner Photo"
                     className="leading-7 text-sm text-gray-600"
                   >
-                    User-Name
+                    Banner Image
                   </label>
                   <input
-                    type="text"
-                    name="username"
+                    type="file"
+                    accept="image/*"
+                    name="profile Photo"
                     className="w-full bg-gray-100 bg-opacity-50 rounded border border-gray-300 focus:border-indigo-500 focus:bg-white focus:ring-2 focus:ring-indigo-200 text-base outline-none text-gray-700 py-1 px-3 leading-8 transition-colors duration-200 ease-in-out"
-                    onChange={(e) =>
-                      setDetails({ ...details, username: e.target.value })
-                    }
+                    // onChange={(e) => setDetails({...details, coverPhoto:{...profilePhoto,chosen: e.target.files[0] }})}
```

Comment:
commented code

Reply:
Thanks for such a detailed and amazing review apurva will work on every single point you asked for thanks again